src/translations/de.json

```diff
@@ -933,12 +933,14 @@
   "StripeConnectAccountForm.createStripeAccountLinkFailed": "Hoppla! Etwas ist schiefgelaufen. Bitte wende dich an den Support.",
   "StripeConnectAccountForm.createStripeAccountLinkFailedWithStripeError": "Hoppla! Etwas ist schiefgelaufen. Stripe hat eine Fehlermeldung zurückgegeben: „{stripeMessage}“",
   "StripeConnectAccountForm.individualAccount": "Ich bin eine Einzelperson",
+  "StripeConnectAccountForm.buttonTitleLimitedAccess": "You can't manage payout details on behalf of user.",
```

Comment:
We don't add these keys for non-english translation files unless they have been already translated. The default setup is going to use keys from en.json as a fallback anyway. (Let's get through this tomorrow.)